packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -70,14 +73,24 @@ const LinkControlSearchInput = ( {
 				__experimentalHandleURLSuggestions={ true }
 			/>
 
-			<Button
-				disabled={ ! value.length }
-				type="reset"
-				label={ __( 'Reset' ) }
-				icon="no-alt"
-				className="block-editor-link-control__search-reset"
-				onClick={ onReset }
-			/>
+			<div className="block-editor-link-control__search-actions">
+				<Button
+					disabled={ ! value.length }
+					type="submit"
+					label={ __( 'Submit' ) }
+					icon="editor-break"
+					className="block-editor-link-control__search-submit"
+				/>
```

Comment:
I'm guessing the position of the loading indicator is hard-coded, but now that we're adding an extra submit button here, there's an overlap which occurs when fetching the link suggestions.

![loading-indicator](https://user-images.githubusercontent.com/1779930/72285795-e4408900-3611-11ea-83e9-1731de9defbd.gif)

Reply:
@karmatosed noticed this as well and there is an issue for it #19544

Reply:
This was my bad. Apologies!